docs/content/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Introduction
+type: docs
+bookToc: false
+---
+# reveal
+
+[![Discord](https://img.shields.io/discord/231173817567805440.svg?colorB=7289DA&label=Discord&logo=Discord&logoColor=7289DA&style=flat-square)](https://discord.gg/nAw95ZV)
+![CircleCI (all branches)](https://img.shields.io/circleci/project/github/HackerHappyHour/reveal.svg?logo=circleci&style=flat-square)
+[![npm](https://img.shields.io/npm/v/@h3/reveal.svg?style=flat-square)](https://www.npmjs.com/package/@h3/reveal)
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg?style=flat-square)](https://github.com/semantic-release/semantic-release)
+[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg?style=flat-square)](http://commitizen.github.io/cz-cli/) 
+[![Greenkeeper badge](https://badges.greenkeeper.io/HackerHappyHour/reveal.svg?style=flat-square)](https://greenkeeper.io/) 
+![node](https://img.shields.io/node/v/@h3/reveal.svg?style=flat-square)
+
+A framework for easily creating beautiful web-based presentations. [Check out the live demo](http://revealjs.com/).
```

Comment:
Do we want to point at this? I'm not sure of your thoughts.

Reply:
I figured good to leave it for now since @HackerHappyHour/reveal is not that much different from the original yet.  http://reveal.js.com is still a great demo of reveal.

Reply:
Works with me 😃